src/core/sort.cc

```diff
@@ -1469,10 +1469,8 @@ py::oobj py::Frame::sort(const PKArgs& args) {
   dt::expr::EvalContext ctx(dt);
 
   if (args.num_vararg_args() == 0) {
-    py::otuple all_cols(dt->ncols());
-    for (size_t i = 0; i < dt->ncols(); ++i) {
-      all_cols.set(i, py::oint(i));
-    }
+    size_t no_cols = 0;
+    py::otuple all_cols(no_cols);
```

Comment:
Was it by accident that `no_cols` changed back to `all_cols`?

Reply:
The previous commit with 
```python
py::otuple no_cols(0)
ctx.add_sortby(py::osort(no_cols))
```
failed during the test. I couldn't open to find out where the error was, the server failed to load the page after many attempts. Since some compilers could be very strict, I assumed `0` could have been interpreted as `int` and failed since `otuple(size_t n)` expects `size_t`. Changing it back to the way it is now worked.

Reply:
I see, I don't think it would fail due to this reason, but you can even omit `0` and just say `py::otuple no_cols;`. By default the tuple will have zero columns.

Reply:
well, I changed it back and it worked this time. :)

Reply:
Yeah, no reason to work. Looks good, will merge now.